core/assets/js/tabbar.js

```diff
@@ -0,0 +1,109 @@
+let activeTabId = ""
+
+export const Tabbar = {
+    mounted() {
+        var initialTab = "tab_"+this.el.getAttribute("initial-tab")
+        this.show(initialTab, true)
+
+    },
+    updated() {
+        this.show(activeTabId, false)
+    },
+
+    show(nextTabId, scrollToTop) {
+        activeTabId = nextTabId
+        var tabs = Array.from(document.querySelectorAll('[id^="tab_"]'));
+
+        if (tabs.filter((tab) => { return tab.id === nextTabId }).length === 0) {
```

Comment:
[Array.prototype.some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) is a bit nicer than filter + length.

Reply:
Changed in my next PR